scripts/utils/create_package_template.py

```diff
@@ -419,6 +419,15 @@ def get_script_directory():
 }
 
 
+def set_placeholder_data(args):
+    for arg in TYPES.get(args.type)["arguments"]:
+        # set placeholder data for unset arguments
+        if not getattr(args, arg):
+            placeholder = f"{arg}-CHANGEME".replace('_', '-')
```

Comment:
I find `CHANGEME` a bit difficult to read, I would use `CHANGE-ME` instead. I think it is also clear to apply the replace only to `arg`
```suggestion
            placeholder = f"{arg.replace('_', '-')}-CHANGE-ME"
```